### ISSUE TITLE
Add URL to repo description

### ISSUE THREAD

**Issue by danqing** [GoogleTrends/data#9]
This repo seems to have a webpage at http://googletrends.github.io/data/, but it's not mentioned anywhere in the repo page. May be a good idea to add it.

**Comment by CloudsAndConcerts** [comment]
Thanks, danqing. I'll ask our developer (who has moved on) if he can resolve and update.

**Comment by danqing** [comment]
Maybe @pirhoo can add you (or someone still at Google) to the org.

**Comment by CloudsAndConcerts** [comment]
No worries, Danquing. Seems this is just a repo we are following, not contributing to.

Best,
--
Arnt Maasø
Associate professor, Dr. art.
Dept. of Media and Communication, Univ. of Oslo
PO BOX 1093 Blindern
0317 Oslo, Norway
Mobile: +47-41420825
www.hf.uio.no/imk/personer/vit/arntm/<http://www.hf.uio.no/imk/personer/vit/arntm/>
Twitter: @arnte

21. feb. 2017 kl. 00.42 skrev Danqing Liu <notifications@github.com<mailto:notifications@github.com>>:


Maybe @pirhoo<https://github.com/pirhoo> can add you (or someone still at Google) to the org.

**Comment by Havenriley** [comment]
Yes thank you. I had a issue about a year ago so I came here your group helped me.
